docs/beds/vibradorm.md

```
@@ -52,30 +64,68 @@ Commands are single bytes written to the command characteristic:
+| M6 | 28 | `0x1C` |
+| Store | 13 | `0x0D` |
 
-Light commands are 3 bytes written to the light characteristic:
+Store-to-memory handshake: `STORE (0x0D)` ×4 → `Memory(slot)` → `STOP` ×4.
+
+### Floor light (CmdLightVMAT, `LIGHT` `0x1529`)
 
 ```text
-[brightness, 0x00, timer]
+[level, 0x00, timerMinutes]
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)